string leg_id
string name_config
int64 pin
string table
string nameRelation
float64 legPinRelationDegree
